src/status_im2/contexts/chat/messages/content/text/view.cljs

```diff
@@ -9,87 +9,106 @@
     [utils.re-frame :as rf]))
 
 (defn render-inline
-  [units {:keys [type literal destination]} chat-id]
-  (case (keyword type)
-    :code
-    (conj units [quo/text {:style (merge style/block (style/code)) :weight :code} literal])
-
-    :emph
-    (conj units [quo/text {:style {:font-style :italic}} literal])
-
-    :strong
-    (conj units [quo/text {:weight :bold} literal])
-
-    :strong-emph
-    (conj units
-          [quo/text
-           {:weight :bold
-            :style  {:font-style :italic}} literal])
-
-    :del
-    (conj units [quo/text {:style {:text-decoration-line :line-through}} literal])
-
-    :link
-    (conj units
-          [quo/text
-           {:style    {:color (colors/theme-colors colors/primary-50 colors/primary-60)}
-            :on-press #(rf/dispatch [:browser.ui/message-link-pressed destination])}
-           destination])
-
-    :mention
-    (conj
-     units
-     [rn/view
-      {:style style/mention-tag-wrapper}
-      [rn/touchable-opacity
-       {:active-opacity 1
-        :on-press       #(rf/dispatch [:chat.ui/show-profile literal])
-        :style          style/mention-tag}
-       [quo/text
-        {:weight :medium
-         :style  style/mention-tag-text}
-        (rf/sub [:messages/resolve-mention literal])]]])
-
-    :edited
-    (conj units
-
-          [quo/text
-           {:weight :medium
-            :style  {:font-size 11 ; Font-size must be used instead of props or the
-                                   ; styles will clash with original message text
-                     :color     (colors/theme-colors colors/neutral-40
-                                                     colors/neutral-50)}}
-           literal])
-    :status-tag
-    (let [community-id (rf/sub [:community-id-by-chat-id chat-id])]
+  [units {:keys [type literal destination]} chat-id style-override]
+  (let [show-as-plain-text? (seq style-override)]
```

Comment:
this could just be the prop value instead, would be slightly nicer as you don't couple style override to this.
e.g
```
(defn render-inline
  [units {:keys [type literal destination]} chat-id show-as-plain-text? ]
  ...
 ```
 
```
(render-inline {...} "chat-id" (seq style-override)) ...
```